src/prop/bvminisat/bvminisat.cpp

```diff
@@ -222,23 +236,29 @@ void BVMinisatSatSolver::toSatClause(const BVMinisat::Clause& clause,
 
 // Satistics for BVMinisatSatSolver
 
-BVMinisatSatSolver::Statistics::Statistics(StatisticsRegistry* registry, const std::string& prefix)
+BVMinisatSatSolver::Statistics::Statistics(StatisticsRegistry* registry,
+                                           const std::string& prefix)
     : d_registry(registry),
-      d_statStarts("theory::bv::"+prefix+"bvminisat::starts"),
-      d_statDecisions("theory::bv::"+prefix+"bvminisat::decisions"),
-      d_statRndDecisions("theory::bv::"+prefix+"bvminisat::rnd_decisions"),
-      d_statPropagations("theory::bv::"+prefix+"bvminisat::propagations"),
-      d_statConflicts("theory::bv::"+prefix+"bvminisat::conflicts"),
-      d_statClausesLiterals("theory::bv::"+prefix+"bvminisat::clauses_literals"),
-      d_statLearntsLiterals("theory::bv::"+prefix+"bvminisat::learnts_literals"),
-      d_statMaxLiterals("theory::bv::"+prefix+"bvminisat::max_literals"),
-      d_statTotLiterals("theory::bv::"+prefix+"bvminisat::tot_literals"),
-      d_statEliminatedVars("theory::bv::"+prefix+"bvminisat::eliminated_vars"),
-      d_statCallsToSolve("theory::bv::"+prefix+"bvminisat::calls_to_solve", 0),
-      d_statSolveTime("theory::bv::"+prefix+"bvminisat::solve_time", 0),
+      d_statStarts("theory::bv::" + prefix + "bvminisat::starts"),
+      d_statDecisions("theory::bv::" + prefix + "bvminisat::decisions"),
+      d_statRndDecisions("theory::bv::" + prefix + "bvminisat::rnd_decisions"),
+      d_statPropagations("theory::bv::" + prefix + "bvminisat::propagations"),
+      d_statConflicts("theory::bv::" + prefix + "bvminisat::conflicts"),
+      d_statClausesLiterals("theory::bv::" + prefix
+                            + "bvminisat::clauses_literals"),
+      d_statLearntsLiterals("theory::bv::" + prefix
+                            + "bvminisat::learnts_literals"),
+      d_statMaxLiterals("theory::bv::" + prefix + "bvminisat::max_literals"),
+      d_statTotLiterals("theory::bv::" + prefix + "bvminisat::tot_literals"),
+      d_statEliminatedVars("theory::bv::" + prefix
+                           + "bvminisat::eliminated_vars"),
+      d_statCallsToSolve("theory::bv::" + prefix
+                         + "bvminisat::calls_to_solve", 0),
+      d_statSolveTime("theory::bv::" + prefix + "bvminisat::solve_time"),
```

Comment:
If you are already changing this code:

Use a common prefix `bvm_prefix` for all the bvminisat statistics.

`std::string bvm_prefix = "theory::bv::" + prefix + "bvminisat::";`
and use it as
`d_statStarts(bvm_prefix + "starts");`

Reply:
That's in the member init so this is not possible.

Reply:
I could move the initialization to the body of the constructor. Want me to do that?

Reply:
Yes that would be an option and nicer to read.

Reply:
Semms that this is not possible since ReferenceStat only has constructors with arguments. I'll leave it as it is.